WalletWasabi.Tests/UnitTests/BitcoinCore/RpcBasedTests.cs

```diff
@@ -48,5 +44,160 @@ public async Task AllFeeEstimateAsync()
 				await coreNode.TryStopAsync();
 			}
 		}
+
+		[Fact]
+		public async Task VerboseBlockInfoAsync()
```

Comment:
Can you add a bit more tests?

Reply:
This is a very general request, there is quite a simple path here, as the structure of a verbose block is quite standard and doesn't change. 
Can you perhaps suggest what you kind of tests would provide more coverage?